projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_configuration_plugin.py

```diff
@@ -59,6 +59,18 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             INGEST_PAYLOAD_PREPROCESS_SEQUENCE and INGEST_METHOD_DEFAULT.
             """,
         )
+        config_builder.add(
+            key="INGEST_IGNORE_PAYLOAD_PREPROCESS_SEQUENCE_FOR_METHOD",
+            default_value=None,
+            description="""A string of coma-separated ingestion methods, for
```

Comment:
Make sure to update first IngesterConfigurationPlugin doc
https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_configuration_plugin.py#L9